components/application-broker/internal/broker/provision_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/pkg/errors"
 	osb "github.com/pmorie/go-open-service-broker-client/v2"
 	"github.com/stretchr/testify/assert"
+	istioversionedclientfake "istio.io/client-go/pkg/clientset/versioned/fake"
```

Comment:
(nit): `istioversionedclientfake` can be `fakeistioversionedclient` or something else to sound good.